code/code/obj/obj_open_container.cc

```diff
@@ -65,12 +65,23 @@ void TOpenContainer::assignFourValues(int x1, int x2, int x3, int x4)
   setCarryVolumeLimit(x4);
 }
 
+int TOpenContainer::getDefaultContainerFlags() const {
+  TObj* prototypeRaw = read_object(objVnum(), VIRTUAL);
+  TOpenContainer* prototype = dynamic_cast<TOpenContainer*>(prototypeRaw);
+  int out = 0;
+  if (prototype != nullptr)
```

Comment:
this isn't what nullptr is for, just use the idomatic `if (prototype)`.

Reply:
Brevity vs explicitness? I'm happy with either, so if it makes you feel better...

Reply:
I think in this case explicitness is actually subtracting information from the code: now I have to assume that we compare against null because treating prototype as a boolean doesn't do what we need. In more complex examples this kind of assumption can lead to wasted time or bugs.